uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/MediaPopupElementView.qml

```diff
@@ -43,74 +44,221 @@ ColumnLayout {
+                    orientation: Qt.Horizontal
+
+                    // Setting a default theme if no color information is provided on the BarSeries
+                    theme: GraphsTheme {
+                        backgroundColor: "white"
+                        plotAreaBackgroundColor: "white"
+                        theme: GraphsTheme.Theme.MixSeries
+                    }
+
+                    axisY: ValueAxis {
+                        max: listModelData.maxValue
+                        min: listModelData.minValue
+                        tickInterval: Math.abs(listModelData.maxValue) > Math.abs(listModelData.minValue)? max / 3 : Math.abs(min) / 3
+                    }
+
+                    BarSeries {
+                        // BarSeries will take owership of the QList<QBarSet*> when we call append. This also applies to PieSeries and QPieSlice for PieChartPopupMediaItem.
+                        // s.a https://github.com/qt/qtgraphs/blob/ef90c963d10a1af30ed523483e9a589235eccef9/src/graphs2d/barchart/qbarseries.cpp#L670-L692
+                        Component.onCompleted: append(listModelData.barSets);
+                    }
+                }
+            }
+
+            Component {
+                id: pieChartComp
+                GraphsView {
+                    height: thisItem.height
+                    width: thisItem.width
+                    marginTop: -25
+                    marginBottom: -25
+                    marginLeft: -25
+                    marginRight: -25
+
+                    // Setting a default theme if no color information is provided on the PieSeries
+                    theme: GraphsTheme {
+                        backgroundColor: "white"
+                        plotAreaBackgroundColor: "white"
+                        theme: GraphsTheme.Theme.MixSeries
+                    }
+
+                    PieSeries {
+                        // PieSeries will take owership of the QList<QPieSlice*> when we call append. This also applies to BarSeries and QBarSet for BarChartPopupMediaItem.
+                        // s.a https://github.com/qt/qtgraphs/blob/08393457537bb45acf3620986039e516f1ac6f2b/src/graphs2d/piechart/qpieseries.cpp#L635-L672
+                        Component.onCompleted: append(listModelData.pieSlices);
+                    }
+                }
+            }
+
+            Component {
+                id: lineChartComp
+                GraphsView {
+                    height: thisItem.height
+                    width: thisItem.width
+                    marginTop: 10
+                    marginBottom: 10
+                    marginLeft: 10
+                    marginRight: 10
+
+                    // Setting a default theme if no color information is provided on the LineSeries
+                    theme: GraphsTheme {
+                        backgroundColor: "white"
+                        plotAreaBackgroundColor: "white"
+                        theme: GraphsTheme.Theme.MixSeries
```

Comment:
Can you explain this? The background will be white, what will the other colors be?

Reply:
Good point. I can reference the Qt doc for `GraphsTheme.Theme.MixSeries`